examples/rust/msm/src/main.rs

```diff
@@ -1,7 +1,9 @@
 use icicle_bn254::curve::{
```

Comment:
maybe add a batch MSM example ?
What about an example running in BM and Large triangle accumulation?

Maybe these should be under benchmarks tho not sure.

Please ignore this comment if you dont think we need this.